test/test.js

```diff
@@ -20,6 +22,12 @@ const SEPARATOR = '\u001F';
 test.beforeEach(async () => pify(exec)(`rm -rf ${dest} ${destUnpacked}`));
 
 test('equals to sample', async t => {
+  // 1482680798652 - datetime for test/fixtures/output.apkg
```

Comment:
Giving variable name `now` is self-explanatory. In the future, I recommend using comments like

```js
const now = 1482680798652; // Dec 25 2016, could be any date
```